coredump_bot.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=CoredumpBot
+
+[Service]
+ExecStart=/home/%u/start_coredump_bot_prod.sh
```

Comment:
Does the bot do anything relative to the current working directory? If yes, I'd set the `WorkingDirectory` too.